pkgs/fc/agent/fc/maintenance/cli.py

```diff
@@ -169,6 +171,9 @@ def run(run_all_now: bool = False, force_run: bool = False):
     The --force-run flag also runs requests in state 'success' again when they are still
     in the queue after a recent system reboot.
 
+    The --online flag (enabled by default) causes interaction with outside
```

Comment:
Is this flag only there for running the maintenances in NixOS tests? Or are there plausible cases for running maintenances offline in the wild?

Reply:
Yes. ;)